### PR TITLE
Setup the roots for Blog App

### PR DESCRIPTION
**Pull request by MeqdamAlqudah**
for each URL you create:

A route.
Action in the correct controller.
A view file.
View files include only placeholders written in plain HTML, e.g:

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  root 'users#index'
+  get "/users/:user_id/posts(.:format)", to: "posts#index"
+  get "/users/:user_id/posts/:id(.:format)", to: "posts#show"
+  get "/users(.:format)", to: "users#index"
+  get "/users/:id(.:format)", to: "users#show"
```
Comment (Cathella) [inline config/routes.rb]: - [ ] [OPTIONAL] You can also opt to use the resources, for example

```
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
end
```

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+development:
+  adapter: postgresql
+  encoding: unicode
+  database: blog_app_development
+  pool: 5
+  username: blog_app
+  password: blog_app
```
Comment (Cathella) [inline config/database.yml:7]: - [ ] [OPTIONAL] I also suggest the use of environment variables to store sensitive data. For example you can make use of the rails inbuilt credentials - `<%= Rails.application.credentials.username %>` then store the values in the encrypted credentials file. To open it use `rails credentials:edit` in the terminal

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @MeqdamAlqudah 👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💪🏻

✅ No linter errors.
✅ You used correct Github Flow.
✅ Professional README.
✅ The project's controllers and routes are well setup.


Cheers and Happy coding!👏👏👏

rosycheeks-mochi-peach


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/database.yml (hunk 2 above)
Comment (MeqdamAlqudah) [reply config/database.yml:7]: thanks done

**Review by TedLivist — COMMENTED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @MeqdamAlqudah,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**